openreview/venue/process/proposed_assignment_pre_process.js

```diff
@@ -4,6 +4,7 @@ async function process(client, edge, invitation) {
   const committeeName = invitation.content.committee_name?.value;
   const { groups } = await client.getGroups({ id: invitation.domain });
   const domain = groups[0];
+  const quota = domain.content.reviewer_assignment_quota?.value
```

Comment:
this quota must be retrieved when `committeeName` is a reviewer or there should be another . This process function is being used for other roles too. 

I would do:
```
domain.content['submission_assignment_max_' + committeeName.toLowerCase()]?.value
```